first-steps.mdx

```diff
@@ -1,137 +1,130 @@
   />
 </Frame>
 
-## 5. Discover your test cases
+When it's done, the AI agent will let you know. You will find your tests on the `test cases` page.
+
+<Frame caption="AI agent discovered test cases and you will find them in the test cases view, screenshot 07/2024">
+  <img
+    src="/images/setup/setup-7-discovery-done.png"
+    alt="AI discovery has finished, screenshot 07/2024"
+  />
+</Frame>
+
+## 6. View your first AI auto-generated test cases
+
+You should have your **first active test cases** generated. The AI agent has discovered the test cases, it auto-generated the test steps and has run them to validate whether they work.
+
+They are `on` which means they will run once you trigger a **test report**.
+
+<Frame caption="First active test cases discovered and auto-generated - screenshot 07/2024">
+  <img
+    src="/images/setup/setup-8-first-tests.png"
+    alt="First active test cases AI discovered and AI auto-generated - screenshot 07/2024 "
+  />
+</Frame>
 
-Drafts of the test cases selected during sign-up can be found in the drafts section. Meanwhile, Octomind will discover test cases on your page according to how your website looks.
+If the agent stumbled over a test step, you might have to lend a hand to review and edit a test step.
 
-<Frame caption="Clicking this button will let our AI agent discover test cases, screenshot 02/2024">
+<Frame caption="Test cases that need review, discovered and auto-generated - screenshot 07/2024">
   <img
-    src="/images/discover.png"
-    width="300"
-    alt="discovery button, screenshot 02/2024"
+    src="/images/setup/setup-8-review-steps.png"
+    alt="Test cases that need review, AI discovered and AI auto-generated - screenshot 07/2024 "
   />
 </Frame>
 
-## 6. Approve your test cases
+## 7. Test your website for the first time
 
-Once our AI agent has finished discovery, you can approve or reject each individual draft to generate its steps.
+Once you have active test cases - they are **on**, you can **run** them. Create your first test report. Click on `run all` button and then `run in the app` and see if they found a bug.
 
-<Frame caption="Approve or deny test cases that octomind discovered - screenshot 02/2024">
+<Frame caption="Run your first test report - screenshot 07/2024">
   <img
-    width="300"
-    src="/images/approve.png"
+    src="/images/setup/setup-9-run-tests.png"
+    alt="Run your first test report - screenshot 07/2024 "
     alt="Approve or deny test cases that octomind discovered - screenshot 02/2024 "
   />
 </Frame>
 
-## 7. Publish your test cases
+## 8. See your first test results
 
-Once our AI agent has finished generating steps, you can publish the test case to disable editing and run it in your [Test Reports](/test-reports)
+You have created your first test report, congratulations! If your test is red we will pinpoint you to step in the app that is broken. Find out more about [test reports](/test-reports.mdx) and [debugging your tests](/debugtopus.mdx).
```

Comment:
I like the thinking, but I'm not sure if "debugging" is what we want first time users to do/ it's quite technical. Maybe just hint them to "create a new test case" as the next mission to do?

Reply:
interesting. I'll send a new commit.